ChangeLog

```diff
@@ -6,6 +6,34 @@
 Fail2Ban: Changelog
 ===================
 
+ver. 0.10.0 (2016/XX/XXX) - gonna-be-released-some-time-shining
+-----------
+
+TODO: list all the fixes/enhancements/etc which came with the PR #1346
+
+- Fixes:
+
+
+- New Features:
+   * IPv6 support:
+     - IP addresses are now handled as objects rather than strings capable for 
+       handling both address types IPv4 and IPv6
+     - iptables related actions have been amended to support IPv6 specific actions
+       additionally
+     - hostsdeny and route actions have been tested to be aware of v4 and v6 already
+     - pf action for *BSD systems has been improved and supports now also v4 and v6
+     - name resolution is now working for either address type
+		 - new conditional section functionality used in config resp. includes:
+       - [Init?family=inet4] - IPv4 qualified hosts only
+       - [Init?family=inet6] - IPv6 qualified hosts only
+
+- Enhancements:
+   * several functionality of class `DNSUtils` moved to new class `IPAddr`, 
+     both classes moved to new module `ipdns`
+   * pseudo-conditional section introduced, for conditional substitution resp. 
+     evaluation of parameters for different family qualified hosts, 
+     syntax `[Section?family=inet6]` (currently use for IPv6-support only).
```

Comment:
there were more in #1346, such as "time with units" definitions, waitfor (IIRC) within tests to speed them up, ...